### ISSUE TITLE
Special Characters breaking input pipe on FreeBSD

### ISSUE THREAD

**Issue by nick-orton** [jaor/xmobar#242]
Starting up Xmonad and Xmobar, everything works as expected.  If browsing with firefox to a page that has a "special" character in it's title, the part of xmobar that displays the content coming from xmonad blanks out.  If xmonad is continued to be used, then it eventually freezes up, but allows the current window to be accessed.  This leads me to believe that the pipe is breaking based on [this](https://wiki.haskell.org/Xmonad/Frequently_asked_questions#XMonad_stops_but_the_current_window_still_responds_to_keys).

A page that will cause the above error is [this](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/getting-started.html).  (There is a non-breaking-space character after the "." in the title.)  Also the github issues page for this repository will also cause the error.  I believe that this is a problem with xmobar, because if I use dzen instead of xmobar in my xmonad.hs, then this problem does not occur.  Killing xmobar and "mod-qing" xmonad gets the system back into a good state.  I don't see anything in any error logs that I was able to find.  If there are any suggestions on how I can get better logging, I would be happy to help out.

Also, I did not see this problem in any linux systems I ran, so this may be a BSD-only problem.
I use "special" character to characterize this because I don't yet know if it's a UTF-8 character problem, or something else.

System:
- OS: FreeBSD 10.2-RELEASE  (x86:64)
- xmobar: 0.23.1  (compiled with UTF_8 flag)
- xmonad: 0.11.1
- xmonad-contrib: 0.11.4
- firefox: 41.0.2,1
  Using xmonad.hs [here](https://github.com/nick-orton/dots/blob/master/xmonad/xmonad.hs)
  Using xmobarrc [here](https://github.com/nick-orton/dots/blob/master/xmonad/xmobarrc)

**Comment by nick-orton** [comment]
I was able to correct this problem on my system.  It seems that FreeBSD does not handle UTF8 by default.  I was able to correct the issue with instructions found [here](https://www.b1c1l1.com/blog/2011/05/09/using-utf-8-unicode-on-freebsd/).  Feel free to close this issue if you want, but I'm keeping it open as Dzen handled this edge case in a way that xmobar did not.  You might be interested in that.
